R/compare_results.R

```diff
@@ -44,7 +44,12 @@ sig_compare <- function(sig1, sig2, metric = c("cosine", "jsd"),
   comparison$y_sig_index <- as.numeric(comparison$y_sig_index)
   comparison <- comparison[order(comparison[[metric_name]], decreasing = TRUE), 
                            ]
```

Comment:
You should try to keep the brackets "]" on the same line. You can automatically format the code in Rstudio.